test/stages/stage2-cp-foundation.tf

```diff
@@ -0,0 +1,9 @@
+# module "gitops_cp_foundation" {
```

Comment:
If this test will always be run on a cluster that has CP4D installed, then this file is not necessary and can be removed.